app/imports/collections/comments.js

```diff
@@ -0,0 +1,19 @@
+import { Meteor } from 'meteor/meteor';
+
+class CommentsCollection extends Mongo.Collection {
+  insert(doc, callback) {
+    doc.createdAt = doc.createdAt || new Date();
+    doc.userId = Meteor.userId();
+    doc.year = doc.year || new Date().getFullYear();
```

Comment:
do we ever actually use this value? i kinda understand the point but there is no way to differenciate between a comment made in november and on in January? or maybe they finish the process all inside this year?